src/Bigcommerce/Api/Client.php

```diff
@@ -491,6 +491,16 @@ public static function deleteProduct($id)
     }
 
     /**
+     * Delete all products.
+     *
+     * @return hash|bool|mixed
+     */
+    public static function deleteProducts()
```

Comment:
Perhaps `deleteAll(Products|Categories|Brands)`? Just so it's explicit that what you're deleting is everything. May not be important though.